envoy-control-tests/src/main/resources/lua_spec/ingress_rbac_logging_spec.lua

```diff
@@ -504,8 +504,9 @@ describe("envoy_on_response:", function()
 
         it("with globally allowed client", function ()
             -- given
-            metadata['envoy.filters.http.rbac']['shadow_engine_result'] = 'allowed'
+            metadata['envoy.filters.http.rbac']['shadow_engine_result'] = 'denied'
```

Comment:
Special client is actually shadow denied, not allowed. You can verify this by checking out master, adding a log line `handle:logInfo` and running `echo2 should allow special client with client identity header over https and log request`